src/Repository/VolunteerRequestRepository.php

```diff
@@ -43,4 +43,35 @@ public function findForReferent(Adherent $referent): array
 
         return $data;
     }
+
+    public function findForMunicipalChief(Adherent $municipalChief): array
+    {
+        if (!$municipalChief->isMunicipalChief()) {
+            return [];
+        }
+
+        $qb = $this->createQueryBuilder('v')
```

Comment:
what about refaco this part to avoid the duplication?
something like that:

```php
$qb = $this->createListQueryBuilder('v');

$where = new Expr\Orx();
foreach ($municipalChief->getMunicipalChiefManagedArea()->getCodes() as $key => $code) {
    $where->add("FIND_IN_SET(:codes_$key, v.favoriteCities) > 0");
    $qb->setParameter("codes_$key", $code);
}

$qb->andWhere($where);

return $this->prepareListData($qb->...->getResult());


```